Adds the Captain's Wardrobe, removes a TON of Captain's Locker bloat items.

ShadeAware opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request
Adds in a new Wardrobe, the Captain's Wardrobe, which holds all cosmetic Captain clothing. Clothing has been moved from the Locker to the vender because of this. 

First time editing a map so expect everything to go wrong. 🙃 
<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->

## Why It's Good For The Game
The Captain's locker is bloated to shit. This helps alleviate the problem and still let people have their fancy mix-and-match outfits.
<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->

## Changelog
:cl:
Add: The Captain's Wardrobe, a vender that holds all cosmetic Captain's clothing.
Remove: Most cosmetic items from the Captain's locker.
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

Commit:
Merge pull request #1 from Citadel-Station-13/master

-

Commit:
Revert "Merge pull request #1 from Citadel-Station-13/master"

This reverts commit cce3a5c5e3bd8dd1bbd94fb5883fe44c414728d1, reversing
changes made to 4d1f0144ef424e5db824ae39c2c44205db7f9062.

Commit:
Revert "Revert "Merge pull request #1 from Citadel-Station-13/master""

This reverts commit b7c774779533d9d32a48f535e8895dcc50c4c909.

Commit:
Bloat-be-Gone!

Commit:
Fixes(?) Icon conflicts

Re

Commit:
(Attempts) to undo faulty map edits

In preperation to do it right the second time around.

Commit:
Revert "(Attempts) to undo faulty map edits"

This reverts commit 44614f5e728b27bc73cd3f9568bfcf44dc69b60b.

Commit:
PROPERLY(?) discards faulty map edits

Commit:
Today I learned that editing maps sucks.

I hope this fixes it.

Ghommie commented on code/modules/vending/wardrobes.dm:
```diff
@@ -400,3 +400,31 @@
+					/obj/item/clothing/head/caphat/beret/white = 1,
+					/obj/item/clothing/under/captainparade = 1,
+					/obj/item/clothing/suit/captunic = 1,
+					/obj/item/clothing/under/rank/captain/femformal = 1, //Fuck your Citadel Edits and fuck locker bloat, we VENDERSTATION now.
```
Remove comment.

Ghommie reviewed (changes requested):
Also merge conflicts

Commit:
Adds Access restrictions to the Captain's Drobe, denied icon

Commit:
No more poggers

Trillby wouldn't stop bitching at me unless I removed the hats and winter coat so lets defeat the purpose of this fucking PR hmm?

Commit:
Revert "No more poggers"

This reverts commit b051ff876df02296ec792b30fa38c37ab34370cc.

Commit:
Fixes the boxstation conflict(?)

Tries to bring the current file in line with the latest rendition of the map.

Commit:
If at first you do not succeed, try again.
